app/_assets/javascripts/app.js

```diff
@@ -683,3 +683,18 @@ $(".closebanner").on("click", function () {
   $(".navbar-v2").addClass("closed");
   localStorage.setItem("closebanner-hackathon", "closebanner");
 });
+
+
+// Tooltips for badges
+jQuery(function () {
+    $('.badge.enterprise')
+      .append( '<div class="tooltip"><span class="tooltiptext">Available with Enterprise subscription</span></div>' );
+    $('.badge.plus')
+      .append( '<div class="tooltip"><span class="tooltiptext">Available with Plus subscription (Konnect Cloud)</span></div>' );
+    $('.badge.free')
+      .append( '<div class="tooltip"><span class="tooltiptext">Available in Free mode (not open-source)</span></div>' );
```

Comment:
This is most likely not actionable feedback in the scope of this PR, but I wanted to share it nonetheless:

It's good that we're clarifying `free != oss` now--I think this fills the "I am running OSS, so why can't I turn on feature xyz? The website says it's free" gap. However, I think the average person will then be left wondering

> well, okay, what does "Free mode" mean?

Anecdotally, I was a long-time Kong user before joining the company a couple months ago, and I honestly had no idea what this entailed until I asked about it internally. Googling terms like "kong enterprise free" and clicking around eventually lead me to [this explanatory blog post](https://konghq.com/blog/kong-gateway-enterprise-2-3-now-generally-available/), but I scrolled right past it several times thinking "nah that can't be the information I'm looking for--that's just a blog random blog post." So IMO there's still a communication gap left to bridge here.

Reply:
I would change it to "Available in Enterprise Free mode (without a license)"

Reply:
^ I like this one. Will go with that.